webapp/src/components/Metrics/PerformanceAnalysisTable.tsx

```diff
@@ -0,0 +1,608 @@
+    ]);
+    setColumnVisibilityModel(Object.fromEntries(columnVisibilityEntries));
+  }, [pipeline, comparedPipeline, metricInfo]);
+
+  const RowLink = (props: RowProps<Row>) => (
+    <Link
+      style={{ color: "unset", textDecoration: "unset" }}
+      to={`/${jobId}/dataset_splits/${selectedDatasetSplit}/performance_overview${constructSearchString(
+        {
+          ...(props.row.id !== OVERALL_ROW_ID && {
+            [selectedMetricPerFilterOption]: [
+              props.row.basePipeline.filterValue,
+            ],
+          }),
+          ...pipeline,
+        }
+      )}`}
+    >
+      <GridRow {...props} />
+    </Link>
+  );
+
+  return errorFetchingBasePipelineData ? (
+    <Typography
+      sx={{ minHeight: 20 }}
+      variant="body2"
+      margin={2}
+      alignSelf="center"
+    >
+      {errorFetchingBasePipelineData.message}
+    </Typography>
+  ) : errorFetchingComparedPipelineData ? (
+    <Typography
+      sx={{ minHeight: 20 }}
+      variant="body2"
+      margin={2}
+      alignSelf="center"
+    >
+      {errorFetchingComparedPipelineData.message}
+    </Typography>
+  ) : (
+    <Paper
+      variant="outlined"
+      sx={{
+        height: "100%",
+        padding: 4,
+        marginTop: 2,
+      }}
+    >
+      <Box marginBottom={1} display="flex" justifyContent="space-between">
+        <Box width={340}>
+          <DatasetSplitToggler
+            availableDatasetSplits={availableDatasetSplits}
+            value={selectedDatasetSplit}
+            onChange={setSelectedDatasetSplit}
+          />
+        </Box>
+        <FormControlLabel
+          label={`Compare Baseline (${
+            config?.pipelines?.[pipeline.pipelineIndex].name
+          } ) with:`}
+          labelPlacement="start"
+          sx={{ gap: 1, paddingRight: 2 }}
+          control={
+            <Select
+              id="compare-pipeline-model-select"
+              variant="standard"
+              value={comparedPipeline ?? "No pipeline"}
+              displayEmpty
+              onChange={({ target: { value } }) =>
+                setComparedPipeline(
+                  typeof value === "number" ? value : undefined
+                )
+              }
+            >
+              <MenuItem value="No pipeline">
+                <em>No pipeline</em>
+              </MenuItem>
+              {config?.pipelines?.map(
+                (pipelineData, index) =>
+                  index !== pipeline.pipelineIndex && (
+                    <MenuItem key={index} value={index}>
+                      {pipelineData.name}
+                    </MenuItem>
+                  )
+              )}
+            </Select>
+          }
+        ></FormControlLabel>
+      </Box>
+      <Paper
+        variant="outlined"
+        sx={{
+          marginTop: 2,
+          padding: 3,
+        }}
+      >
+        <Table
+          sx={{
+            "& .MuiDataGrid-cell": {
+              borderBottom: "none",
+            },
```

Comment:
I found an easy solution to the column borders not being continuous when the header is an icon, with `DATA_GRID_PROPS_DEFAULT_VALUES` `from "@mui/x-data-grid"`:
```suggestion
            },
            "& .MuiDataGrid-columnHeader": {
              // Fix line-height not affecting non-text headers like icons
              height: DATA_GRID_PROPS_DEFAULT_VALUES.headerHeight,
            },
```

Reply:
Nice!.. I made some CSS to display the borders without breaks in between. So, what is this "DATA_GRID_PROPS_DEFAULT_VALUES"? is it a const?

Reply:
Yes, a const `DATA_GRID_PROPS_DEFAULT_VALUES` `from "@mui/x-data-grid"`

Reply:
By adding this "DATA_GRID_PROPS_DEFAULT_VALUES.headerHeight" CSS property, I could still see the breaks between vertical borders next to the headers. When I investigated, I noticed that height defaults to "26px" for icon headers, whereas it's 56px for others. I assume it's because of the autoHeight?

Reply:
I tested my suggestion and it worked. Maybe you have an issue with other changes you have locally?

Reply:
I don't think it's required now. so we can resolve this

Reply:
I think the borders must be added back to the headers. But good if this CSS is not necessary anymore.